motif-docs/pages/guides/config/using-code.mdx

```diff
@@ -441,6 +441,10 @@ Environment variables can be configured two ways
     - Example: `"engine": "postgres"`
     - Supported values: `"postgres", "mysql", "mariadb"`
 
+    `applyChangesImmediately: boolean`
```

Comment:
I think we should add how this is reflected in the UI (which is something I've recently changed)

Reply:
i.e.:

<img width="770" alt="Pasted_Image_22_11_22__20_14" src="https://user-images.githubusercontent.com/21973269/203312820-380aca84-0794-4c99-a296-77867483354a.png">